bin/lib/zopen-build

```diff
@@ -1140,29 +1137,31 @@ deleteDuplicateEntries()
 {
   value=\$1
   delim=\$2
-  echo "\$value" | awk -v RS="\$delim" '!(\$0 in a) {a[\$0]; printf("%s%s", col, \$0); col=RS; }'
+  echo "\$value\$delim" | awk -v RS="\$delim" '!(\$0 in a) {a[\$0]; printf("%s%s", col, \$0); col=RS; }' | sed "s/\${delim}$//"
 }
 
 _CEE_RUNOPTS="\$_CEE_RUNOPTS FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+# Remove spaces in between brackets
+_CEE_RUNOPTS="\$(echo "\$_CEE_RUNOPTS" | awk '{gsub(/\([ \t]*/, "("); gsub(/[ \t]*\)/, ")"); gsub(/[ \t]*,[ \t]*/, ","); print}')"
```

Comment:
this is ok for now, but some runtime options have strings that you wouldn't want to remove spaces from

Reply:
I see, USRHDLR(lmname,"") seems to be one example. As long as there is not comma or () within the quotes, then it should be ok.